MapleServer2/Commands/Game/InfoCommands.cs

```diff
@@ -18,77 +18,132 @@ public InfoCommands()
                 "commands"
             };
             Description = "Give informations about all commands";
+            Usage = "/commands";
         }
 
         public override void Execute(GameCommandTrigger trigger)
         {
-            List<CommandBase> commandList = CommandManager.CommandsByAlias.Values.ToList();
-            StringBuilder stringBuilder = new();
+            List<IGrouping<string, CommandBase>> commandList = CommandManager.CommandsByAlias.Values.GroupBy(x => x.Aliases[0]).ToList();
+            StringBuilder stringBuilder = new StringBuilder("Commands:\n");
 
-            foreach (CommandBase command in commandList)
+            foreach (IGrouping<string, CommandBase> commandGroup in commandList)
             {
-                foreach (string alias in command.Aliases)
+                string aliases = string.Empty;
+                if (commandGroup.First().Aliases.Skip(1).Any())
                 {
-                    stringBuilder.Append(CommandHelpers.Color(CommandHelpers.Bold(alias), Color.DarkOrange) + " ");
+                    aliases = $" [{string.Join(", ", commandGroup.First().Aliases.Skip(1))}]";
                 }
+                stringBuilder.Append(CommandHelpers.Color(CommandHelpers.Bold(commandGroup.First().Aliases.First() + aliases), Color.DarkOrange) + " ");
```

Comment:
Also you can make the strings method `Color` and `Bold` in `CommandHelper` an extension.
```suggestion
                string message = commandGroup.First().Aliases.First() + aliases;
                stringBuilder.Append(message.Bold().Color(Color.DarkOrange) + " ");
```

Then in commandHelper `make the method public with `this string message` as the first parameter

Reply:
Suuure